docker-for-windows/troubleshoot.md

```diff
@@ -281,8 +290,8 @@ Virtualization needs to be enabled in the BIOS. The steps to do so are Vendor
 specific, but typically the BIOS option is called `Virtualization Technology
 (VTx)` or similar.
 
-Once Hyper-V is enabled, it shows up as such on "Turn Windows features on or
-off".
+From the START menu enter in "Turn Windows features on or
```

Comment:
Replace "START" with "Start".
You also have too many "and" in this sentence. Can you update it to use some commas instead?

Reply:
Sure